port_definition.go

```diff
@@ -0,0 +1,23 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package marathon
+
+// PortDefinition provides metadata about a port
+type PortDefinition struct {
+	Port     int    `json:"port,omitempty"`
```

Comment:
@timoreimann nice catch on that - you are correct about port 0 being a special value, will push a fix for this line to the branch.